### ISSUE TITLE
Feature request: Can use `tqdm` module when `model.fit(verbose=2)`?

### ISSUE THREAD

**Issue by imhuay** [keras-team/keras#10274]
When use `model.fit(verbose=2)`, it will output the result after finishing one epoch. But it will cannot see the process of loss.

Can use `tqdm` module replace current implement?
